### DISCUSSION TITLE
WebDav

### DISCUSSION THREAD

**Discussion by patrowawrick** [sircharlo/meeting-media-manager#127] [Q&A] [answered]
Bonjour à tous

Je ne maîtrise pas cet technique mais j'aimerais bien l'utiliser. Quel serveur Cloud gratuit permet d'accéder aux fichiers par WebDav ? Est-ce que ça fonctionne avec Google Drive, OneDrive ? Sinon avec quoi ? Merci de votre aide technique sur le sujet. Avec tout mon amour fraternel.

_Google Translation :
I do not master this technique but I would like to use it. Which free cloud server allows to access files via WebDav ? Does it work with Google Drive, OneDrive ? Otherwise with what? Thank you for your technical help on the subject. With all my brotherly love._

Patrick W
France

**Comment by sircharlo** [comment, marked as answer]
Bonne question! 
Je vais ajouter un peu de documentation là-dessus, avec exemples. J'ai trouvé au moins deux sites gratuits qui offrent la synchronisation WebDAV, pour ceux qui (contrairement à moi) n'ont pas de serveur dedié à leur disposition.
Un autre amélioration prévue pour la mise à jour de fin-octobre! 😉

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/41bea79670550742ed7f12ff54feea54329d90c5 and https://github.com/sircharlo/jw-meeting-media-fetcher/commit/2e3022b0364abceeb437735c8b849cd7e881ae3b

**Comment by patrowawrick** [comment]
Merci beaucoup ! J'attends avec impatience ton document ;o)

**Reply by sircharlo** [reply]
Nouvelle mise à jour parue dans #129! Si tu regardes la [documentation mise à jour](https://github.com/sircharlo/jw-meeting-media-fetcher#congregation-level-media-syncing-%EF%B8%8F), j'espère qu'elle t'aidera. Sinon n'hésites pas à me poser des questions ici.

**Comment by patrowawrick** [comment]
😘😉
